tower-http/src/services/fs/serve_dir/tests.rs

```diff
@@ -496,11 +496,16 @@ async fn read_partial_rejects_out_of_bounds_range() {
         .unwrap();
     let res = svc.oneshot(req).await.unwrap();
 
-    assert_eq!(res.status(), StatusCode::RANGE_NOT_SATISFIABLE);
+    assert_eq!(res.status(), StatusCode::PARTIAL_CONTENT);
```

Comment:
So if all bytes get returned, does it still make sense for the status code to be 'partial content'?

Reply:
I did actualy consider that and if I understand the spec correctly, it does https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/206
> The HTTP 206 Partial Content success status response code indicates that the request has succeeded and the body contains the requested ranges of data, as described in the [Range](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Range) header of the request.